homeassistant/components/openweathermap/const.py

```diff
@@ -146,7 +146,7 @@
     "rainy": [300, 301, 302, 310, 311, 312, 313, 500, 501, 520, 521],
     "snowy": [600, 601, 602, 611, 612, 620, 621, 622],
     "snowy-rainy": [511, 615, 616],
-    "sunny": [800],
+    "sunny/clear-night": [800],
```

Comment:
remove this line, conditions must represent constants for weather card

Reply:
But isn't this more descriptive? Both constants are possible for this weather code so IMO it would make more sense to have a combination of both as the key here. 
Something similar is done in the NWS integration:
https://github.com/home-assistant/core/blob/8b998365a46a3a3db5b46dddfce9b65025d91c46/homeassistant/components/nws/const.py#L65-L67

Reply:
Yeah, I think you right.